gravitee-apim-console-webui/package.json

```diff
@@ -20,11 +20,16 @@
     "@asyncapi/web-component": "1.2.26",
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2024-45811   
**Check Name:** vite: server.fs.deny is bypassed when using `?import&raw`   
**Severity:** MEDIUM   
**Fixed Version:** 5.4.6, 5.3.6, 5.2.14, 4.5.4, 3.2.11, 5.1.8   
**Reachable Path(s) Found:** No  
**Description:** Vite a frontend build tooling framework for javascript. In affected versions the contents of arbitrary files can be returned to the browser. `@fs` denies access to files outside of Vite serving allow list. Adding `?import&raw` to the URL bypasses this limitation and returns the file content if it exists. This issue has been patched in versions 5.4.6, 5.3.6, 5.2.14, 4.5.5, and 3.2.11. Users are advised to upgrade. There are no known workarounds for this vulnerability.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2024-45811